src/components/TaskManager.tsx

```diff
@@ -0,0 +1,15 @@
+import TaskForm from "./TaskForm";
+import TaskList from "./TaskList";
+
+function TaskManager(): JSX.Element {
+  return (
+    <main>
```

Comment:
There's a `<main>` here that is not present in TaskList page, that is, it will dissapear when navigating to TaskList. You must put a `<main>` tag in TaskList or move it to App.tsx, so you don't have to write it on every page.